lib/boring.js

```diff
@@ -11,7 +11,7 @@
 module.exports = function() {
   var args = process.argv.slice(2);
   var result = {
-    _ = []
```

Comment:
weird how did this ever work? property initialization syntax maybe

Reply:
I'm not sure we're even using this file anymore. boring is now its own package that you imported in another file.

Reply:
Yeah we probably aren't.